tensorflow/contrib/seq2seq/python/ops/attention_wrapper.py

```diff
@@ -1182,6 +1191,10 @@ def __init__(self,
             "cell_input_fn must be callable, saw type: %s"
             % type(cell_input_fn).__name__)
 
+    if attention_layer_size is not None and attention_layer is not None:
```

Comment:
If ... is not None == ... is not None:
  Really

Reply:
Err.. then Raise

Reply:
I think the condition is correct here. Additionally, both can be `None` which is a valid case.